lib/SILGen/SILGenBuiltin.cpp

```diff
@@ -1604,6 +1604,16 @@ static ManagedValue emitCreateAsyncTask(SILGenFunction &SGF, SILLocation loc,
       return emitOptionalNone(ctx.TheExecutorType);
     }
   }();
+  ManagedValue taskExecutorConsuming = [&] {
+    if (options & CreateTaskOptions::OptionalEverything) {
+      return nextArg().getAsSingleValue(SGF);
+    } else {
+      auto theTaskExecutorProto = ctx.getProtocol(KnownProtocolKind::TaskExecutor);
+      assert(theTaskExecutorProto && "Could not find TaskExecutor");
+      return emitOptionalNone(theTaskExecutorProto->getDeclaredExistentialType()
+                                  ->getCanonicalType());
+    }
+  }();
```

Comment:
The crashes you're seeing likely come from here, because TaskExecutor isn't in older _Concurrency libraries.

Reply:
Thanks! Managed to find a solution: https://github.com/apple/swift/pull/74000/commits/c310d083582f49dea2af79778405bbb86efe91aa we don't need the specific right type here since we're just passing `nil`. John suggested just using the builtin task executor type and it seems to work. 

Thank you!